src/mots/export.py

```diff
@@ -32,20 +35,19 @@ def escape_for_rst(value: str) -> str:
 
 
 def format_paths_for_rst(
-    value: list, indent: int = DEFAULT_LIST_TABLE_INDENT, directory: Directory = None
+    value: List[str], indent: int, directory: Directory = None
 ) -> str:
     """Escape and format a path string (e.g. for includes or excludes)."""
     config = directory.config_handle.config
     try:
         searchfox_enabled = config["export"]["searchfox_enabled"]
     except KeyError:
         searchfox_enabled = False
-        searchfox_base_url = None
+
+    if searchfox_enabled:
+        searchfox_base_url = f"https://searchfox.org/{config['repo']}/search?q=&path="
```

Comment:
Nit: should we have the searchfox URL in `settings` in the same way as the PMO URL?